src/locales/en.json

```diff
@@ -1,14 +1,14 @@
 {
   "nav": {
-    "item_files": "Files",
+    "item_files": "Drive",
     "item_recent": "Recent",
     "item_shared": "Shared by me",
     "item_activity": "Activity",
     "item_trash": "Trash",
     "item_settings": "Settings"
   },
   "breadcrumb": {
-    "title_files": "Files",
+    "title_files": "Drive",
```

Comment:
I would keep *Files* in that context.
But if you change the value, consider changing the key as well.
And to depend less on the translation, maybe `breadcrumb.title` is explicit enough.